app/src/main/java/com/vaiwork/playlistmaker/ui/media/fragment/PlaylistsFragment.kt

```diff
@@ -15,6 +21,11 @@ class PlaylistsFragment : Fragment() {
     private val playlistsViewModel: PlaylistsViewModel by viewModel()
     private var binding: FragmentPlaylistsBinding? = null
```

Comment:
🍏 Используй lateinit, чтобы потом все время не проверять binding на null или использовать !!
```suggestion
    private lateinit val binding: FragmentPlaylistsBinding
```
https://kotlinlang.org/docs/properties.html#late-initialized-properties-and-variables